### DISCUSSION TITLE
A Full-width Banner Pic for Page?

### DISCUSSION THREAD

**Discussion by tsavpyn** [withastro/starlight#1991] [Feature Requests]
### What version of `starlight` are you using?

0.24.1

### What is your idea?

**A banner picture** for the page that is **full-sized with the content area width** instead of insert an independent image,
and *display the title below the banner picture*
The configuration (such as image url and maybe something else) of this feature should be customizable in the `frontmatter`. 

### Why is this feature necessary?

This helps sorting the documents and giving directions to users to tell which the document is. 
Also, this could make documents more individual and recognizable. 
I believe that a **full-sized banner picture** supported to be set in `frontmatter` in every `md/mdx` file will be a great idea. 

### Participation

- [ ] I am willing to submit a pull request for this feature.

**Comment by lorenzolewis** [comment]
This should be achievable with making a custom page with the Starlight layout:

https://starlight.astro.build/guides/pages/#custom-pages
https://starlight.astro.build/guides/pages/#using-starlights-design-in-custom-pages

**Comment by tsavpyn** [comment]
Thanks to the devs for their patience. 
I understand that this feature is not so necessary and urgent at the moment. 
Happy building! The discussion is now closed. :)
